src/app/main-nav/main-nav.component.html

```diff
@@ -0,0 +1,69 @@
+<header class="nhsuk-header" role="banner">
+  <a class="nhsuk-skip-link" [routerLink] [fragment]="skipLinkSelector">Skip to main content</a>
+  <div class="nhsuk-width-container nhsuk-header__container">
+    <div class="nhsuk-header__logo">
+      <a class="nhsuk-header__link nhsuk-header__link--service " routerLink="./" aria-label="NHS Revalidation">
+        <svg class="nhsuk-logo" xmlns="http://www.w3.org/2000/svg" role="presentation" focusable="false"
```

Comment:
can we not reference the svg directly from the assets folder?

Reply:
yup, but in all honesty. I'm not referencing this main nav in application. This is the template given by nhs frontend. Using material navigation as it works out of the box

Reply:
ah so is this template not being used then? maybe we can remove it?

Reply:
removed. done